src/main/java/io/stargate/sgv2/jsonapi/api/configuration/CommandObjectMapperHandler.java

```diff
@@ -1,13 +1,34 @@
 package io.stargate.sgv2.jsonapi.api.configuration;
 
+import com.fasterxml.jackson.core.JsonParser;
 import com.fasterxml.jackson.databind.DeserializationContext;
 import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonDeserializer;
 import com.fasterxml.jackson.databind.deser.DeserializationProblemHandler;
 import com.fasterxml.jackson.databind.jsontype.TypeIdResolver;
 import io.stargate.sgv2.jsonapi.exception.ErrorCode;
 import io.stargate.sgv2.jsonapi.exception.JsonApiException;
+import java.io.IOException;
 
 public class CommandObjectMapperHandler extends DeserializationProblemHandler {
+
+  @Override
+  public boolean handleUnknownProperty(
+      DeserializationContext ctxt,
+      JsonParser p,
+      JsonDeserializer<?> deserializer,
+      Object beanOrClass,
+      String propertyName)
+      throws IOException {
+    if (deserializer.handledType().toString().endsWith("CreateCollectionCommand$Options")) {
+      throw ErrorCode.INVALID_CREATE_COLLECTION_OPTIONS.toApiException(
+          "No option \"%s\" found as createCollectionCommand option", propertyName);
```

Comment:
This is helpful: but even more helpful I think would be listing 2 existing options in addition?
Something like:

```
"No option \"%s\" found as createCollectionCommand option (known options: \"indexing\", \"vector\")",
```

Reply:
(not sure if `"vectorize"` should be included; it is technically allowed, but not publicly supported.

Reply:
solved, thanks